File: articles/algorithmic-harm-error-and-institutional-responsibility/typescript/harm_risk_score.ts
const errorLikelihood = 0.34;
const severity = 0.92;
const exposure = 0.78;
const contestability = 0.42;
const harmRisk = errorLikelihood * severity * exposure * (1.0 - contestability);
console.log(`harm_risk_score=${harmRisk.toFixed(4)}`);

File: articles/algorithmic-harm-error-and-institutional-responsibility/rust/harm_risk_score.rs
fn main() {
    let error_likelihood = 0.34;
    let severity = 0.92;
    let exposure = 0.78;
    let contestability = 0.42;
    let harm_risk = error_likelihood * severity * exposure * (1.0 - contestability);
    println!("harm_risk_score={:.4}", harm_risk);
}
